.github/workflows/release-publish.yml

```diff
@@ -84,3 +94,28 @@ jobs:
           script: |
             cd /service/deploy/fosslight/
             bash /service/deploy/fosslight/deploy.sh
+
+  deploy_image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/fosslight:${{ github.event.release.tag_name }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
Combine jobs(`release-publish.yml` / `deploy-image-to-docker-hub.yml`) into one workflow